git-send-email.perl

```diff
@@ -411,6 +416,7 @@ sub read_config {
 			$smtp_encryption = 'ssl';
 		}
 	}
+	return;
```

Comment:
Face F

On Friday, June 10, 2016, Johannes Schindelin <notifications@github.com>
wrote:

> In git-send-email.perl
> <https://github.com/git/git/pull/251#discussion_r66608127>:
>
> > @@ -411,6 +416,7 @@ sub read_config {
> >  			$smtp_encryption = 'ssl';
> >  		}
> >  	}
> > +	return;
>
> This return is also unnecessary.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/git/git/pull/251/files/c6659b11db7db4e087a12a20954f4afe9ac13c6f#r66608127>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADGmfXf632Db5_7TBnRwk7psGGhccHaks5qKV8IgaJpZM4IwZoQ>
> .
>


-- 
Plz excuse typos, sending from my handheld.